### ISSUE TITLE
Jellyfin Client 1.10.1 Crash on MacOS Ventura 13.6.3

### ISSUE THREAD

**Issue by MorrisC2010** [jellyfin/jellyfin-media-player#668]
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               Jellyfin Media Player [26978]
Path:                  /Applications/Jellyfin Media Player.app/Contents/MacOS/Jellyfin Media Player
Identifier:            tv.jellyfin.player
Version:               1.10.1 (1.10.1)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2024-05-27 13:29:15.9112 -0500
OS Version:            macOS 13.6.3 (22G436)
Report Version:        12
Anonymous UUID:        CDC2283B-F2A8-1BFB-675E-E14BB97633C5

Sleep/Wake UUID:       1C396365-E0C5-4E1C-A4C4-85C687594A53

Time Awake Since Boot: 25000 seconds
Time Since Wake:       349 seconds

System Integrity Protection: enabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000

Termination Reason:    Namespace DYLD, Code 4 Symbol missing
Symbol not found: _$sSo13NSWindowLevela6AppKit01_cdD23NumericRawRepresentableACMc
`Referenced from: <2E98933C-5ADA-3FCB-97DE-95A140BA166D> /Applications/Jellyfin Media Player.app/Contents/Frameworks/libmpv.2.dylib (built for macOS 14.0 which is newer than running OS)`
Expected in:     <956685F7-4174-3135-911D-CF52948CE09A> /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit
(terminated at launch; ignore backtrace)

**Comment by Sp9nky** [comment]
same here on 13.6.7

**Comment by rohankandwal** [comment]
I got stuck with the ignore and download buttons not working so thought of upgrading to the latest version 1.10.1, this version crashes with same error as reported in this bug. However, if i download the old version 1.10.0 and move that to application directory, I get same crash in the older version as well. So, now I can't use new or old version.

**Comment by mjpieters** [comment]
This applies to macos 12.7.4 as well.

**Comment by mjpieters** [comment]
Right, as per [this comment](https://github.com/jellyfin/jellyfin-media-player/pull/628#issuecomment-2112713188), 1.10.x requires macos 14 or newer when running on Apple silicon, so this is 'by design'. Sigh.

**Comment by siggisigg** [comment]
I have the same issue, does building from source work as a work around ?

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
